java/src/org/openqa/selenium/grid/node/local/LocalNode.java

```diff
@@ -82,6 +82,8 @@
 import org.openqa.selenium.grid.data.Session;
 import org.openqa.selenium.grid.data.Slot;
 import org.openqa.selenium.grid.data.SlotId;
+import org.openqa.selenium.grid.data.SlotMatcher;
```

Comment:
I don't think changes in this class are needed anymore.

Reply:
updated